### PR TITLE
React tutorial

### PR DESCRIPTION
**Pull request by AlexisJW**
The following were implemented:

-  Persisting data using local storage.
-  Routing using react-router.
-  Rendering dynamic links.
-  react hooks, react-router, and styling using a module have been implemented in this project.
- Deploy the project with Vercel

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            style={({ isActive }) => ({
+              color: isActive ? 'red' : undefined,
+            })}
+          >
+            {link.text}
+          </NavLink>
+        </li>
+      ))}
+    </ul>
+  </nav>
```
Comment (hansderly) [inline src/components/Navbar.js:24]: [⚠️] Please considering adding style to your navigation

**Review by hansderly — APPROVED** [review]
# Status: Approved ✅
Hi @AlexisJW   👋 , 


NoddingYesGIF


Good job so far!
There is nothing left to say, it's time to merge it 👏🎉
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

✅ Descriptions are very descriptive
✅ Linters checks are passing
✅ Use GitFlow
✅ Meaningful commit messages.


## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navbar.js (hunk 1 above)
Comment (AlexisJW) [reply src/components/Navbar.js:24]: Thank you @hansderly, I'll do it!

**Review by hansderly — APPROVED** [review]
# Status: Approved ✅
Hi @AlexisJW   👋 , 


NoddingYesGIF


Good job so far!
There is nothing left to say, it's time to merge it 👏🎉
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

✅ Descriptions are very descriptive
✅ Linters checks are passing
✅ Use GitFlow
✅ Meaningful commit messages.


## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**